.cfignore

```diff
@@ -1 +1,2 @@
 *.swp
+e2e
```

Comment:
why do we need to ignore e2e for cf?

Reply:
This is what gets pushed to cloud.gov. Since cloud.gov buildpack does not have cypress, javascript, node support, there is no reason to push it. We should probably be more aggressive in what we don't push, but this is a fine step.